sci-physics/vgm/vgm-4.4.ebuild

```diff
@@ -27,7 +27,8 @@ RDEPEND="
 	root? ( sci-physics/root:=[-root7] )
 	geant4? ( >=sci-physics/geant-4.10.03 )"
 DEPEND="${RDEPEND}
-	doc? ( app-doc/doxygen[dot] )"
+	doc? ( app-doc/doxygen[dot] )
+	test? ( sci-physics/geant-vmc[g4root] )"
 RESTRICT="
 	!geant4? ( test )
 	!root? ( test )"
```

Comment:
You may also want to include `!test? ( test )`, given tests won't probably work with USE=-test.

Reply:
Oh yes, of course! 
Sorry, I was under the wrong impression that this was implied, but of course it's not. 
Added it and re-pushed.